teachopencadd/talktorials/T038_protein_ligand_interaction_prediction/talktorial.ipynb

```diff
@@ -0,0 +1,995 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #32.                (filename[:-4], pdb_to_graph(os.path.join(os.path.join(self.folder_name, "proteins", filename)))) for filename in os.listdir(os.path.join(self.folder_name, "proteins"))
</pre><p>pathlib (see above)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/volkamerlab/teachopencadd/pull/290/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='24'/>